src/views/MobileDownloadLinks.tsx

```diff
@@ -0,0 +1,107 @@
+import styled from 'styled-components';
+
+import { ButtonShape, ButtonType } from '@/constants/buttons';
+import { STRING_KEYS } from '@/constants/localization';
+
+import { useStringGetter } from '@/hooks/useStringGetter';
+
+import { IconName } from '@/components/Icon';
+import { IconButton } from '@/components/IconButton';
+import { Link } from '@/components/Link';
+import { VerticalSeparator } from '@/components/Separator';
+
+export const MobileDownloadLinks = ({ withBadges }: { withBadges?: boolean }) => {
+  const stringGetter = useStringGetter();
+
+  const androidUrl = document
```

Comment:
nit: I wonder if we can leverage `getMobileAppUrl` from https://github.com/dydxprotocol/v4-web/blob/10d540188136caeeb0d9978d548d0fc7daa84015/src/views/dialogs/MobileDownloadDialog.tsx

Reply:
Nice! Moved selector logic to a hook and used on `AccountMenu`, `MobileDownloadDialog` and `MobileDownloadLinks`